_specs/ecip-_.md

```diff
@@ -0,0 +1,103 @@
+---
+ecip: TBD
+title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
+author: Mr. Meows D. Bits
+discussions-to: TBD
+status: WIP
+type: Meta
+created: 2019-11-29T14:17:42-05:00
+resolution: TBD
+---
+
+### Abstract
+
+Establishing _Fork_ type ECIPs, and their standards and processes.
+
+#### TL;DR
+
+Make ECIPs with `Type` _Fork_ a thing. _Fork_ ECIPs should follow this process:
+- A "shell" Fork ECIP is opened, fitting the [template provided](TODO). This ECIP __does not initially include `Activation Block Number` or `Features` specifications__.
+- While in `WIP` or `Draft` status, one or more change sets are proposed against this document (eg. via Github Pull Requests) __modifying BOTH `Activation Block Number` and `Features` specifications__.
```

Comment:
Probably also states that those PRs should not be merged until the community consensus is reached?

Reply:
I would prefer not to mention or even begin to tangle with that idea. I'm interested here only in tangible, describable mechanics.

(Including that idea might provoke questions like "What is that?" ...Which I would prefer to leave as a dubious project for another very very brave soul to tackle in a different place at a different time.)

Reply:
@meowsbits Yeah I understand. But what should editors do when multiple changesets are present? I just think no matter what we choose, the process specification must state this clearly.

Reply:
Let me think about this. I will sleep on it and get back to you tomorrow.

Reply:
I'm leaning toward keeping this document as "mechanic" as possible; to describe only "the physics" of ECIP process events. This proposal is intended to be structural, and rooted in material patterns, and to mostly just formalize an existing practice with a few minor (but important, in my opinion) tweaks. ... :smile_cat: I want to help program and code the event-based system; but not to provide the type definitions or factory for providing those events... yet )). 

Pull requests are getting merged already, somehow. ECIPs are changing status already, somehow. Human "consensus"/decision-making (either as a potential specification or as practical tradition) has important impacts for this repository's specifications including, but certainly not limited to, this one. 

If you feel strongly a need to specify and document how "human consensus" does or should happen in a given case, then I would suggest we do so in a separate concern.

I am -- as always -- open to further discussion, especially if you would (please :heart_decoration: ) propose a change set we can use a concrete reference point.

Reply:
(And let's leave this open -- and see if anyone else has any other ideas ;)

Reply:
@sorpaas See https://github.com/ethereumclassic/ECIPs/pull/224#discussion_r352338257 for a draft meant to show a change that might fit your intention here.